deployment/environment/memory/node.go

```diff
@@ -364,10 +368,24 @@ func NewNode(
 		UnrestrictedHTTPClient:   &http.Client{},
 		RestrictedHTTPClient:     &http.Client{},
 		AuditLogger:              audit.NoopLogger,
+		RetirementReportCache:    retirement.NewRetirementReportCache(lggr, db),
 	})
 	require.NoError(t, err)
 	keys := CreateKeys(t, app, chains, solchains, aptoschains)
 
+	nodeLabels := make([]*ptypes.Label, 1)
```

Comment:
do we know if these 2 nodeType labels are common across domains?

Reply:
Well, nobody else is using the `nodeType` label, so we get to set the pattern.